### PR TITLE
BLOGS-1341 - Add Welsh translations

### PR DESCRIPTION
**Pull request by mattgreenham**
https://jira.dev.bbc.co.uk/browse/BLOGS-1341

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/BaseController.php [hunk 1]
```diff
@@ -27,6 +27,9 @@
     /** @var mixed[] */
     protected $otherIstatsLabels = [];
 
+    /** @var string */
```
Comment (votemike) [inline src/Controller/BaseController.php]: `/** @var string|null */`

File: translations/blogs/cy.po [hunk 2]
```diff
@@ -262,6 +270,10 @@ msgstr "Mae'n ddrwg gennym, rydyn ni wedi methu dod o hyd i’r dudalen"
 msgid "page_title_archive_month"
 msgstr "Pob cofnod ar gyfer"
 
+#. English: "There are no results".
+msgid "no_results"
```
Comment (votemike) [inline translations/blogs/cy.po]: Where was this used in the end?

Comment (mattgreenham) [reply translations/blogs/cy.po]: AuthorIndex, AuthorShow, TagIndex, TagShow

**Pull request merged**